lib/ansible/modules/cloud/vmware/vmware_vswitch_info.py

```diff
@@ -111,8 +112,8 @@ def serialize_pnics(vswitch_obj):
             pnics.append(pnic.split("-", 3)[-1])
         return pnics
 
-    def gather_vswitch_facts(self):
-        """Gather vSwitch facts"""
+    def gather_vswitch_info(self):
+        """Gather vSwitch info"""
         hosts_vswitch_facts = dict()
```

Comment:
should we also rename this one?

Reply:
Fine for me. Though as I said, I don't mind if more internal renames are done in a follow-up PR, so we can get the module rename done eventually :)